descqa/shear_test.py

```diff
@@ -50,8 +50,11 @@ def __init__(self,
                  **kwargs):
         #pylint: disable=W0231
 
-        plt.rcParams['font.size'] = 9
-
+        #plt.rcParams['font.size'] = 9
```

Comment:
Remove unused code instead of just commenting it out. If it must be kept, provide a reason inline.
```suggestion
```